### PR TITLE
Version changes, fixes and unit tests

### PR DESCRIPTION
**Pull request by soerenmartius**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,7 +25,7 @@ endif
 
 ifndef NOCOLOR
   GREEN  := $(shell tput -Txterm setaf 2)
-  YELLOW := $(shell tput -Txterm setaf 3)
+  YELLOW := $(shell tput -Txterm seaf 3)
```
Comment (thiesen) [inline Makefile:28]: was `setaf` changed in this line by mistake? I couldn't find anything regarding a `seaf` capname :sweat_smile:

Comment (soerenmartius) [reply Makefile:28]: yes that a mistake and should be fixed - good catch!

Comment (soerenmartius) [inline Makefile:28]:
```suggestion
  YELLOW := $(shell tput -Txterm setaf 3)
```

File: README.tfdoc.hcl [hunk 2]
```diff
@@ -147,7 +147,7 @@ section {
         readme_example = <<-END
           projects_access = [{
             project = "your-project-id"
-            role    = "roles/editor"
+            roles    = ["roles/editor"]
```
Comment (mariux) [inline README.tfdoc.hcl:150]:
```suggestion
            roles   = ["roles/editor"]
```

File: README.tfdoc.hcl [hunk 3]
```diff
@@ -159,11 +159,11 @@ section {
...
         }
 
-        attribute "role" {
+        attribute "roles" {
           required    = true
-          type        = string
+          type        = set(string)
           description = <<-END
-            The role that should be applied.
+            The set of roles that should be applied.
```
Comment (mariux) [inline README.tfdoc.hcl:166]:
```suggestion
            A set of roles that should be applied.
```

File: README.tfdoc.hcl [hunk 4]
```diff
@@ -189,11 +189,11 @@ section {
...
```
Comment (mariux) [inline README.tfdoc.hcl:196]:
```suggestion
            A set of roles that should be applied.
```

File: README.tfdoc.hcl [hunk 5]
```diff
@@ -219,11 +219,11 @@ section {
...
```
Comment (mariux) [inline README.tfdoc.hcl:226]:
```suggestion
            A set of roles that should be applied.
```

File: Makefile (hunk 1 above)
Comment (mariux) [inline Makefile:28]: please revert this change ;)

**Review by mariux — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (soerenmartius) [reply Makefile]: resolved

**Review by mariux — APPROVED** [review]
lgtm
